src/core/subgraph-helper/index.ts

```diff
@@ -36,8 +37,21 @@ const subgraphHelperAbi = [
 // Returns Subgraph Helper contract instance without signer
 export const getSubgraphHelperInstance = (chain: Chain): Contract => {
   try {
-    return new ethers.Contract(SUBGRAPH_HELPER_ADDRESS, subgraphHelperAbi);
+    // @todo Need to fix the references library to allow chainId as ArbSepolia has some issues
+    return new ethers.Contract(contracts[42161].SubgraphHelper.address, subgraphHelperAbi);
```

Comment:
we should pass chain here instead of 42161

Reply:
Its not possible currently because there's a typo in references library. The address for subgraph helper for ArbSepolia does not exist. I've added it as a todo, it will be fixed in the next version of references lib update